lib/logstash/inputs/http_poller.rb

```diff
@@ -42,6 +51,20 @@ def register
 
     @logger.info("Registering http_poller Input", :type => @type, :schedule => @schedule, :timeout => @timeout)
 
+    @url_field = ecs_select[disabled: "[#{metadata_target}][request][url]", v1: "[url][full]"]
+    @method_field = ecs_select[disabled: "[#{metadata_target}][request][method]", v1: "[http][request][method]"]
+    @host_field = ecs_select[disabled: "[#{metadata_target}][host]", v1: "[host][hostname]"]
```

Comment:
When ECS compatibility is enabled, these three fields no longer respect `metadata_target`, despite them being metadata about how this particular plugin retrieved the event. I don't think we have enough information about usage to be this opinionated. If the event contains any of these fields, we stomp on them with something that is _less_ likely to be meaningful.